src/ConductorSharp.Toolkit/Service/ScaffoldingService.cs

```diff
@@ -0,0 +1,286 @@
+
+            var taskDefinitions = await metadataService.GetAllTaskDefinitions();
+
+            var taskCollectionBuilder = new StringBuilder();
+            foreach (var taskDefinition in taskDefinitions)
+            {
+                var taskClass = await CreateTaskClass(taskDefinition);
+
+                if (taskClass != null)
+                    taskCollectionBuilder.Append(taskClass);
+            }
+
+            var taskTemplate = EmbeddedFileHelper.GetLinesFromEmbeddedFile("~/Templates/TaskCollectionTemplate.default");
+            var taskCollection = taskCollectionBuilder.ToString();
+            taskTemplate = taskTemplate.Replace("{{namespace}}", config.BaseNamespace + ".Tasks");
+            taskTemplate = taskTemplate.Replace("{{taskCollection}}", taskCollection);
+            var dir = config.Destination + Path.DirectorySeparatorChar + "Tasks";
+
+            Directory.CreateDirectory(dir);
+
+            File.WriteAllText(dir + Path.DirectorySeparatorChar + "Tasks.scaff.cs", taskTemplate);
+
+            var workflowTemplate = EmbeddedFileHelper.GetLinesFromEmbeddedFile("~/Templates/WorkflowCollectionTemplate.default");
+            var workflowCollection = workflowCollectionBuilder.ToString();
+            workflowTemplate = workflowTemplate.Replace("{{namespace}}", config.BaseNamespace + ".Workflows");
+            workflowTemplate = workflowTemplate.Replace("{{workflowCollection}}", workflowCollection);
+            var workflowDir = config.Destination + Path.DirectorySeparatorChar + "Workflows";
+
+            Directory.CreateDirectory(workflowDir);
+
+            File.WriteAllText(workflowDir + Path.DirectorySeparatorChar + "Workflows.scaff.cs", workflowTemplate);
+        }
+
+        public async Task<string> CreateWorkflowClass(WorkflowDefinition workflowDefinition)
+        {
+            var lines = EmbeddedFileHelper.GetLinesFromEmbeddedFile("~/Templates/WorkflowTemplate.default");
+
+            string name = SnakeCaseUtil.ToPascalCase($"{workflowDefinition.Name}_V{workflowDefinition.Version}").Trim();
+            string note = null;
+            if (LangUtils.MakeValidMemberName(name, "A", out name))
+            {
+                note = "The autogenerated name has been changed because it is invalid C# member name";
+            }
+            //name = char.ToUpperInvariant(name[0]) + name[1..];
+
+            string workflowDescription = workflowDefinition.Description;
+            string[] labels = null;
+
+            try
+            {
+                var descriptionObject = JObject.Parse(workflowDescription);
+                workflowDescription = descriptionObject.SelectToken("description")?.Value<string>();
+                var labelsObject = descriptionObject.SelectToken("labels");
+
+                if (labelsObject is JArray labelsArray)
+                {
+                    labels = labelsArray.ToObject<string[]>();
+                }
+            }
+            catch (Exception) { }

```

Comment:
We should add a log here with something like Failed parsing description for workflow WORKFLOWNAME

Reply:
Done